runtime/devnet/src/lib.rs

```diff
@@ -630,9 +630,9 @@ impl Contains<RuntimeCall> for SafeModeWhitelistedCalls {
 
 parameter_types! {
 	pub const EnterDuration: BlockNumber = 4 * HOURS;
-	pub const EnterDepositAmount: Balance = UNITS;
+	pub const EnterDepositAmount: Option<Balance> = None;
```

Comment:
Any reason for preferring `Option::None` over `Balance::zero()`?

Reply:
According to [the docs](https://github.com/paritytech/polkadot-sdk/blob/release-polkadot-v1.1.0/substrate/frame/safe-mode/src/lib.rs#L89) `None` is a sane default, while `Balance::zero()` is not specified as such and I suspect it'd allow anyone to enter into safe mode at zero cost.

Reply:
I see! thanks for clarifying